docs/book/src/reference/versions.md

```diff
@@ -120,12 +120,15 @@ Cluster API maintainers might decide to support API versions longer than what is
 
 | API Version | Status         | Supported Until                                                                  |
 |-------------|----------------|----------------------------------------------------------------------------------|
-| v1beta1     | Supported      | at least 9 months or 3 minor releases after a newer API version will be released |
-| v1alpha4    | Not served (*) | EOL since 2023-12-05 - v1.6.0 release date                                       |
-| v1alpha3    | Not served (*) | EOL since 2023-07-25 - v1.5.0 release date                                       |
+| v1beta2     | Supported      | at least 9 months or 3 minor releases after a newer API version will be released |
+| v1beta1     | Deprecated     | Deprecated since CAPI v1.11, EOL planned for v1.14, Aug 26                       |
```

Comment:
Let's be slightly more explicit. Will v1.14 still support v1beta1 or not?
(I think it won't)

same for the table below in l.165

Reply:
PTAL